topics/sequence-analysis/tutorials/ncbi-fcs/tutorial.md

```diff
@@ -0,0 +1,193 @@
+questions:
+- Are the sequences in a genome assembly contaminant-free?
+objectives:
+- Learn how to screen a genome assembly for adaptor and vector contamination.
+- Learn how to screen a genome assembly for non-host organism contamination.
+time_estimation: 90M
+key_points:
+- Contamination can arise from multiple sources and occur at multiple stages of a genome assembly project.
+- Contamination cleanup is essential for reliable comparative genomics analyses.
+contributors:
+- tvedtees
+
+---
+
+
+# Introduction
+
+<!-- This is a comment. -->
+
+The National Center for Biotechnology Information (NCBI) performs contamination screening of genome assemblies submitted to the archival GenBank. Advances in genome sequencing have accelerated the production of genome assemblies and their submission to public databases, necessitating high-performance screening tools. Since contaminants can lead to misleading conclusions about the biology of the organism in question (e.g. gene content, gene evolution, etc.), ideally contamination screening should be performed after the initial contig assembly and prior to downstream genome analyses. 
+
+NCBI has released a publicly-available Foreign Contamination Screen (FCS) tool suite to detect contaminants from various sources and produce a cleaned sequence set. This tutorial provides a quick example of two current FCS tools: FCS-adaptor identifies synthetic sequences used in library preparation, and FCS-GX {% cite Astashya2024 %} identifies sequences from foreign organisms assigned to discordant taxonomies compared to the user-declared source organism.
+
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Retrieving the data
+
+FCS operates on assembled genome sequences and is not intended for use on raw reads. The following tutorial uses an assembled genome from yeast (*Saccharomyces cerevisiae*) with contaminants artificially inserted into the genome. The first step is to retrieve the genome FASTA. 
+
+## Upload the genome FASTA from NCBI FCS GitHub
+
+The following steps provide instructions to upload the test dataset into your Galaxy instance.
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> **Step 1:** Create a new history for this tutorial
+>
+>    > {% snippet  faqs/galaxy/histories_create_new.md %}
+>
+> **Step 2:** Copy the datasets URLs into clipboard.
+>     Click on the copy button in the right upper corner of the box below.
+>    ```
+>    https://zenodo.org/records/10881551/files/FCS.combo.test.fa
+>    ```
+>
+> **Step 3:** Upload `fasta.gz` dataset into Galaxy
+>    > {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    > {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+{: .hands_on}
+
+## Confirm dataset upload
+
+Your data should look like this:
+
+![Galaxy History with Uploaded Data](../../images/ncbi-fcs/fcs-galaxy-1.jpg "Figure 1. Galaxy History with Uploaded Data.")
```

Comment:
```suggestion
![Galaxy History with Uploaded Data](../../images/ncbi-fcs/fcs-galaxy-1.jpg "Galaxy History with Uploaded Data.")
```

When you supply a figure caption (the `".."` after the filename) the figure numbering is automatic, this would result in a "Figure 1. Figure 1. Galaxy Histories..."